external/netfx-conflicts/netfx-conflicts.depproj

```diff
@@ -9,6 +9,7 @@
     <PackageTargetFallback>netstandard1.1;netstandard1.2;netstandard1.3</PackageTargetFallback>
     <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
     <NugetRuntimeIdentifier>None</NugetRuntimeIdentifier>
+    <BinPlaceNETFXRuntime>false</BinPlaceNETFXRuntime>
```

Comment:
Instead we should just disable all binplacing for this.

How is it that CI didn't catch this?